builtin/providers/github/resource_github_team.go

```diff
@@ -43,31 +45,29 @@ func resourceGithubTeamCreate(d *schema.ResourceData, meta interface{}) error {
 func resourceGithubTeamRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*GithubClient).client
 
-	_, err := getGithubTeam(d, client)
+	t, err := getGithubTeam(d, client)
 	if err != nil {
 		d.SetId("")
 		return nil
 	}
-	//TODO need to update Description but go-github doesn't expose it
+	d.Set("name", t.Name)
```

Comment:
Curious. I hadn't thought about this. `Name` is a `ForceNew` field for us, but I wonder if you can update a team name in Github and still keep the same ID. Probably. 
Can you do a PR for this? We can then remove the `ForceName` from the `name` attribute in the schema, because we can then update the internal state to match a new name.